CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Changelog
 
+## Version 0.29.0
+
+### Summary
+
+#### Update Pauli Twirling
+Thanks to @purva-thakre for updating Mitiq's PT functions, clarifying that PT is a noise tailoring technique and for consolidating utilities to be shared between PT and other techniques.
+
+```py
+from mitiq.pt.pt import pauli_twirl_circuit
+
+pauli_twirl_circuit(circuit)
+```
+
+#### Classical Shadows
+
+Top-level functions and tests for classical shadows estimation are now available in Mitiq.
+Congrats @min-li on completing the main functionality for this technique!
```

Comment:
Lets add a short note that documentation on this method is not yet out, but coming soon!

Reply:
Good idea, added in latest commit.